### PR TITLE
LBRY add

### PR DESCRIPTION
**Pull request by VladimirLevv**

### CODE REVIEW

**Comment by akhavr** [comment]
Please, fix conflicts

File: projects/lbry.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+App Developer: [Alex Liebowitz](../people/alex_liebowitz.md)  
+Curator: [Reilly Smith](../people/reilly_smith.md)  
+## Business Model
+LBRY is the first digital marketplace to be controlled by the market’s participants rather than a corporation or other 3rd-party. It is the most open, fair, and efficient marketplace for digital goods ever created, with an incentive design encouraging it to become the most complete.  
+## Contacts
+[GitHub](https://github.com/lbryio/lbry)  
+[Bitcointalk](https://bitcointalk.org/index.php?topic=1541268.0)   
+[Facebook](https://www.facebook.com/lbryio/)  
+[Twitter](https://twitter.com/lbryio?lang=en)  
+[Linkedin]()
```
Comment (akhavr) [inline projects/lbry.md]: Why empty fields?

**Comment by VladimirLevv** [comment]
solved conflicts & removed empty fields

**Comment by akhavr** [comment]
Closes #163

**Pull request merged**